boilerplate-simple-api/internal/modules/hello/controller/method_create_hello_test.go

```diff
@@ -0,0 +1,44 @@
+package hello_controller_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	hello_request "simple-api/internal/modules/hello/models/request"
+	pkg_http_wrapper "simple-api/pkg/http/wrapper"
+
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateHello(t *testing.T) {
+	srv := setupTest()
+
+	t.Run("Create Hello Http Test 1", func(t *testing.T) {
```

Comment:
deskripsi test yg ngegambarin ekspektasi unit test itu apa ya

Reply:
contoh:
- success - should return expected result
- failed - should return error from usecase